content/index.rst

```diff
@@ -86,11 +87,47 @@ Software for Humans
     'open-source/plone'
     ]
 
+.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. callmacro:: content.macros.j2#rst
+  :tag: 'start'
+
+Open Source Leadership
+----------------------
+
+Experts in Code Architecture & Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Founded by a `Django core developer`_
+and the `creator of Susy`_ responsive-layout framework,
+open-source software has always been central
+to the OddBird vision.
+Whether we're building tools for developers like ourselves,
+or third-party consumers,
+we keep the needs of real people
+at the center of everything we do.
+
+.. callmacro:: content.macros.j2#link_button
+  :url: '/contact/'
+
+  Hire us for a workshop or conference talk
+
+.. _Django core developer: /birds/#bird-carl
+.. _creator of Susy: /birds/#bird-miriam
+
+.. callmacro:: content.macros.j2#rst
+  :tag: 'end'
+.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. callmacro:: content.macros.j2#get_quotes
   :slug: 'open-source/susy'
   :index: 1
 
-.. ---------------------------------
+
+
+.. -------------------------------------
+.. callmacro:: content.macros.j2#divider
+
+.. ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. callmacro:: content.macros.j2#rst
   :tag: 'start'
 
```

Comment:
``` .. image:: /static/images/pages/jssass.png
    :class: extend-left size-quarter
    :class: extend-left size-quarter```

Would it make more sense to remove the class `size-quarter` so we can edit the size in CSS for this specific image so it isn't weird when small?

Or maybe override the clear and make it float even when small? 

I know the bigger picture is what you are talking about as far as small extended images ... but they default to full width when you are at certain sizes before extending, the size-quarter is more likely the problem. We probably should be use the secret macro to fix this.